internal/resources/webform/webform.js

```diff
@@ -2435,48 +2449,67 @@ window.initGRPCForm = function(services, invokeURI, metadataURI, debug, headers)
     const clearHistory = () => {
         if (confirm('Are you sure you wish to delete all history? This action is permanent and cannot be undone.')) {
             history = [];
+            preloadedHistory = [];
             onHistoryChange();
         }
     }
 
+    const download = (filename, text) => {
+        var element = document.createElement('a');
+        element.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(text));
+        element.setAttribute('download', filename);
+
+        element.style.display = 'none';
+        document.body.appendChild(element);
+
+        element.click();
+
+        document.body.removeChild(element);
+    }
+
+    const saveHistory = () => {
+        let toSave = history
+        if (preloadedHistory.length > 0 && confirm('Include pre-loaded (gray) items?')) {
```

Comment:
I don't think this is necessary: 1) the JSON is not that hard to edit, to remove any unwanted items and 2) a user could also manually curate by first clearing history, then issuing RPCs, and then saving.

This seems to be the only reason that you have `preloadHistory` as a separate var from `history`. And I think the change is easier to read and follow without it.

Reply:
I guess if we need to split out the pre-loaded history for other reasons, this can stay.

But I like the idea of calling them "example" instead of "pre-loaded items".

Also, instead of just gray background, would it make more sense for this to be very explicit and be in a separate table with a heading that read "Examples"?

Reply:
Completely removed preloadedHistory